realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -87,7 +87,9 @@ void BaseRealSenseNode::publishTopics()
     setupDevice();
     setupPublishers();
     setupStreams();
-    publishStaticTransforms();
+
+    // NOTE: This is done in Miso software now.
```

Comment:
Why can't we just remove this all together? Is the team really used to this or something?

Reply:
This is effectively a comment in the code to make it more readable. This may not even be permanent, we don't really know.